shaders/MembraneWaterRipple.gdshader

```diff
@@ -0,0 +1,182 @@
+shader_type spatial;
+render_mode blend_mix, depth_prepass_alpha, cull_back, diffuse_lambert;
+
+uniform vec4 water_color : source_color = vec4(0.0, 0.0, 0.0, 0.02); // Almost completely transparent background

```

Comment:
Our shaders should also use our C# naming convention so this should be converted to use pascalCase naming convention rather than snake_case. Also the trailing comments should be changed to be comments on the line before what they are commenting on to match Thrive style.